CONTRIBUTING.md

```diff
@@ -173,6 +173,47 @@ The `legacy_foo` arg was removed
 :::
 ```
 
+## Style and idioms
+
+For the most part, we just accept whatever the code formatters do, so there
+isn't much style to enforce.
+
+Some miscellanous style, idioms, and conventions we have are:
+
+### Markdown/Sphinx Style
```

Comment:
Do you want to specify on `*` over `+` for unordered lists? How about indentation?

Reply:
Yeah, I'd be ok with that. I was looking at markdown formatters last week, actually. All the ones I found use `-` for lists (which I don't like but w/e)

Reply:
I prefer `+` and 4-spaces myself, but don't care enough to fight.

I have no reason for `+` but my reason for 4 spaces is that it matches with ordered listed up to 99:

```markdown
1.  foo
    + bar
    + bar
2.  baz with a long list item that ends with a
    line continuation
99. foobar
```

Reply:
I'm going to omit the list thing for now. I tried a few autoformatters for markdown, and they're all very opinionated and not very configurable. e.g. they rewrite all the lists. I'd rather we just pick an auto formatter and use it.

I'm going to keep the line length part, though. There's a few places in our docs where the lines are way way over 100 chars and it drives me nuts.